docs/guides/multipart.md

```diff
@@ -16,7 +16,7 @@ The structure of a multipart request is typically consistent of:<br>
 Crow supports multipart requests and responses though `crow::multipart::message`.<br>
 A message can be created either by defining the headers, boundary, and individual parts and using them to create the message. or simply by reading a `crow::request`.<br><br>
 
-Once a multipart message has been made, the individual parts can be accessed throught `mpmes.parts`, `parts` is an `std::vector`, so accessing the individual parts should be straightforward.<br>
+Once a multipart message has been made, the individual parts can be accessed throughout `mpmes.parts`, `parts` is an `std::vector`, so accessing the individual parts should be straightforward.<br>
```

Comment:
@luca-schlecker @Mr-Destructive This is acutally meant to be "through", I'll make another commit fixing it.

Reply:
I'm sorry I interpreted the meaning wrongly.

Reply:
Oh, I am sorry... I looked at it and agreed with him because I thought it was a list... My bad.

Reply:
no worries, it was my typo to begin with 😅. I'll fix it asap